exercises/bracket-push/example.rkt

```diff
@@ -0,0 +1,23 @@
+#lang racket
+
+(provide is-paired?)
+
+(define (match a b)
+  (ormap (curry equal? (list a b))
+         (list '(#\( #\)) '(#\[ #\]) '(#\{ #\}))))
+
+(define (one-step xs)
+  (cond [(> 2 (length xs)) xs]
+        [(match (car xs) (cadr xs)) (one-step (cddr xs))]
+        [else (cons (car xs) (one-step (cdr xs)))]))
+
+(define (is-paired-helper xs)
+  (let ([new (one-step xs)])
+    (cond [(empty? new) #t]
+          [(= (length xs) (length new)) #f]
+          [else (is-paired-helper new)])))
+
+(define (is-paired? xs)
```

Comment:
Is there a more descriptive name that can be used for the arguments to these functions?